ansible/inventory/group_vars/apps.yaml

```diff
@@ -74,25 +74,25 @@ messaging:
     contact_email: test@email.com
 
 compute:
-  key_pair_name: sandpit-key
+  key_pair_name: densikat-sandpit
   inbound_from_bastion_host_security_group: sg-a5ac6cc2
 
 s3:
   data_bucket_name: aem-stack-builder
 
 library:
-  aem_aws_stack_provisioner_version: 1.1.1
-  aem_orchestrator_version: 1.0.0
+  aem_aws_stack_provisioner_version: 1.1.2
+  aem_orchestrator_version: 1.0.1
   aem_password_reset_version: 1.0.1
   oak_run_version: 1.4.15
 
 publish_dispatcher:
   stack_name: aem-publish-dispatcher-stack
   instance_profile: sandpit-aem-roles-stack-PublishDispatcherInstanceProfile-V9GSTTG597O
   instance_type: t2.micro
-  min_size: 1
-  desired_capacity: 2
-  max_size: 2
+  min_size: 2
+  max_size: 4
+  cpu_scaling_threshold: 80
```

Comment:
Please parameterise this one, threshold tend to be different between users.

Reply:
And please also restore desired capacity as it is used by Orchestrator for adjusting the number of publish instances to match publish-dispatcher ones.